test/definitions/simple_workflow_v0.1.0.json

```diff
@@ -6,6 +6,28 @@
   "version_minor": 1,
   "version_micro": 0,
   "tags": ["speech_to_text"],
+  "rights": [
+    {
+      "action": "view",
+      "groups": ["user"]
```

Comment:
update rights regarding the previous version of the same workflow.
Great to test is as a non user group the filter is correct ...

Reply:
Can you explain what you mean by: "Great to test is as a non user group the filter is correct"